Library/Source/RuntimeUWP.cpp

```diff
@@ -6,8 +6,13 @@
 
 namespace babylon
 {
-    RuntimeUWP::RuntimeUWP(ABI::Windows::UI::Core::ICoreWindow* window, const std::string& rootUrl)
-        : Runtime { std::make_unique<RuntimeImpl>(window, rootUrl) }
+    RuntimeUWP::RuntimeUWP(ABI::Windows::UI::Core::ICoreWindow* window, LogCallback&& callback)
+        : RuntimeUWP{ window, ".", std::move(callback) }
```

Comment:
The string parameter was original defaulting to `{}`. Now, it is `"."`. Which is the right one?

Reply:
I don't think that value is supposed to be meaningful for UWP, so I just changed this to match `RuntimeAndroid` for consistency's sake.

Reply:
Not sure how best to test this, and now agree that it doesn't really belong in this PR anyway, so just reverting.